src/react-recoil-hooks-testing-library.test.tsx

```diff
@@ -78,7 +78,7 @@ describe('react-recoil-hooks-testing-library', () => {
     };
 
     const { result } = renderRecoilHook(useRecoilTestHookWithContext, {
-      wrapper: ({ children }) => (
+      wrapper: ({ children }: PropsWithChildren<{}>) => (
```

Comment:
See my comment below

```suggestion
      wrapper: ({ children }) => (
```